zh/access-dashboard.md

```diff
@@ -0,0 +1,171 @@
+---
+title: 访问 TiDB Dashboard
+summary: 介绍如何在 Kubernetes 环境下访问 TiDB Dashboard
+category: how-to
+---
+
+# TiDB Dashboard 指南
+
+`TiDB Dashboard` 是 TiDB 4.0 专门用来帮助观察与诊断整个 TiDB 集群的可视化面板，你可以在 [TiDB Dashboard](https://github.com/pingcap-incubator/tidb-dashboard) 了解详情。 本篇文章将介绍如何在 Kubernetes 环境下访问 TiDB Dashboard。
+
+## 快速上手
+
+> **注意：**
+>
+> 以下教程仅为演示如何快速 `TiDB Dashboard`，请勿在生产环境中直接使用以下方法。 
+
+`TiDB Dashboard` 目前在 4.0 版本中已经内嵌在了 PD 组件中，你可以通过以下的例子在 Kubernetes 环境下快速部署一个 4.0.0-rc 版本的 TiDB 集群。
+
+```yaml
+apiVersion: pingcap.com/v1alpha1
+kind: TidbCluster
+metadata:
+  name: tidb
+spec:
+  storageClassName: ${storageClass}
+  timezone: UTC
+  pvReclaimPolicy: Delete
+  imagePullPolicy: IfNotPresent
+  pd:
+    image: pingcap/pd:v4.0.0-rc
+    replicas: 1
+    requests:
+      storage: "1Gi"
+    config: {}
+  tikv:
+    config: {}
+    image: pingcap/tikv:v4.0.0-rc
+    replicas: 1
+    requests:
+      storage: "1Gi"
+  tidb:
+    enableAdvertiseAddress: true
+    image: pingcap/tidb:v4.0.0-rc
+    imagePullPolicy: IfNotPresent
+    replicas: 1
+    service:
+      type: ClusterIP
+    config: {}
+    requests:
+      cpu: 1
+```
+
+当集群创建完毕时，你可以通过以下指令将 `TiDB Dashboard` 暴露在本地机器:
+
+{{< copyable "shell-regular" >}}
+
+```shell
+kubectl port-forward svc/tidb-pd -n ${namespace} 2379:2379
+```
+
+然后在浏览器中访问 `http://localhost:2379/dashboard` 即可访问到 `TiDB Dashboard` 
+
+## 通过 Ingress 访问 TiDB Dashboard
+
+你可以通过 `Ingress` 来将 TiDB Dashboard 暴露服务，从而在集群外通过 http/https 的方式访问服务。 你可以通过 [Ingress](https://kubernetes.io/zh/docs/concepts/services-networking/ingress/) 了解更多。
+
+```yaml
```

Comment:
This also might cause confusion. Please give more instructions before listing the yaml file.